media/base/media_switches.cc

```diff
@@ -78,6 +78,9 @@ const char kForceWaveAudio[] = "force-wave-audio";
 // http://crbug.com/259165 for more details.
 const char kTrySupportedChannelLayouts[] = "try-supported-channel-layouts";
 
+// Use RSSDK for video capture.
+const char kUseRsVideoCapture[] = "use-rs-video-capture";
```

Comment:
I think you'll also need a build system flag in addition to the runtime command line one: as is, the code will break the content shell builds, as libpxc is pulled in by Crosswalk, which is not checked out in the content shell bots.